### ISSUE TITLE
Confusion about the difference between the code and the paper

### ISSUE THREAD

**Issue by weiqingcai** [takashiishida/pconf#2]
Hi @takashiishida，I have read your paper, but some confusion arises when reading the code.
  In your code, the formula that needs to be minimized is  : `loss = torch.sum(-model(inputs)+ logistic * 1. / confidence)` and `logistic = -1. * negative_logistic(-1. * model(inputs))` 
  While in your paper ,the formula is 
![formula](https://user-images.githubusercontent.com/10105321/70504187-259bcf80-1b60-11ea-92f4-4dc9837b2c11.png)
  It looks that the minimized formula differences between paper and code

**Comment by takashiishida** [comment]
Thank you for reading the paper.  When a logistic loss function is used, the following can be used: \ell_L(g) - \ell_L(-g) = -g.  Then, the empirical risk used in the paper can be reformulated into the one that is used in the implementation.  I will add a comment to the code to clarify this.

**Comment by weiqingcai** [comment]
Thank you very much for your reply. If I didn't understand it wrong, you mean 
![formula2](https://user-images.githubusercontent.com/10105321/70600845-b8557080-1c2b-11ea-9dee-c460858fd87e.png).
But i can only get ![formula3](https://user-images.githubusercontent.com/10105321/70601658-88a76800-1c2d-11ea-8902-50c3af7a7c4f.png)

Another question is, when I replace the formula in the code`logistic = -1. * negative_logistic(-1. * model(inputs)) loss = torch.sum(-model(inputs)+ logistic * 1. / confidence)` with the formula in the paper`loss = torch.sum(negative_logistic(model(inputs)) + negative_logistic(-model(inputs))*(1.-confidence)/confidence)`, the results are different. What is the reason?Why the results obtained by directly optimizing the formulas in the paper are poor？
